src/actions/identityActions.js

```diff
@@ -240,11 +241,28 @@ export function identityWithdraw({ amountToWithdraw, withdrawTo }) {
 	return async function(dispatch, getState) {
 		try {
 			const account = selectAccount(getState())
+			const {
+				identityBalanceDai,
+				totalIdentityBalanceDai,
+			} = account.stats.formatteds
+			let sweepTxns
+			if (
```

Comment:
do you need to check if it's less than `totalIdentityBalanceDai`? Shouldn't some other part of the code validate this (cc @IvoPaunov )?